app/views/event/participations/_actions_show_sac_cas.html.haml

```diff
@@ -10,3 +10,10 @@
 
 - if @event.course? && can?(:create, @event) && entry.roles.map(&:class).any?(&:leader?)
   = action_button(t(".attach_key_data_sheet"), group_event_key_data_sheets_path(@group, @event, {participation_ids: entry.id}), :file_pdf, method: :post)
+
+- if can?(:leader_settlement, entry.model)
+  - if entry.roles.any? { |role| Event::Course::LEADER_ROLES.include?(role.type) && role.self_employed } # has any self employed leader role
+    = action_button(t('.create_course_leader_invoice_button'),
+                  nil,
+                  :plus,
```

Comment:
Ich fände hier `:file-pdf` icon besser als das `:plus` (wird ja nichts angelegt in der DB)